patches/headless-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/headless/BUILD.gn b/headless/BUILD.gn
+index e0ec74566e0474012ac660fc8929853219f5fcc7..67a046ea4135e74ae0e44f4e0bffc015585ec3bf 100644
+--- a/headless/BUILD.gn
++++ b/headless/BUILD.gn
+@@ -96,7 +96,7 @@ repack("resource_pack_strings") {
+   if (is_chrome_branded) {
+     sources += [ "${root_gen_dir}/components/strings/components_google_chrome_strings_en-US.pak" ]
+   } else {
+-    sources += [ "${root_gen_dir}/components/strings/components_chromium_strings_en-US.pak" ]
++    sources += [ "${root_gen_dir}/components/strings/components_${branding_path_product}_strings_en-US.pak" ]
```

Comment:
same as above

Reply:
I'll submit a CL tomorrow for these cases.

Reply:
Done. https://crrev.com/c/4882660